src/common/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { PropsWithChildren, ReactNode } from 'react';
+import classNames from 'classnames';
+
+import styles from './Checkbox.module.css';
+
+type Props = PropsWithChildren<{
+  onChange: () => void;
+  label?: string | ReactNode;
+  className?: string;
+}>;
+
+function Checkbox(props: Props) {
+  return (
+    <div className={classNames(styles.container, props.className)}>
+      <input
+        className={styles.checkbox}
+        type="checkbox"
+        onChange={props.onChange}
+      />
+      <span className={styles.label}>{props.label}</span>
```

Comment:
I missed this the first time, sorry! This `span` should be a `label` associated with the input.

Reply:
Span changed to label & added name prop as required.